src/components/Navbar/styles.css

```diff
@@ -0,0 +1,128 @@
+.nav-wrapper {
+  padding: 16px 24px;
+  position: sticky;
+  top: 0;
+  z-index: 99;
+  backdrop-filter: blur(50px);
+  background: rgba(0, 0, 0, 0.1);
+}
+
+.nav-content {
+  max-width: 1300px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 1rem 0;
+  margin: 0 auto;
+}
+
+.logo {
+  width: 172px;
+  height: auto;
+}
+
+.nav-content ul {
+  display: flex;
+  align-items: center;
+  gap: 1.5rem;
+  list-style: none;
+}
+
+.nav-content li {
+  margin: 0 24px;
+}
+
+.menu-item {
+  text-decoration: none;
+  font-size: 0.9rem;
```

Comment:
same here, use Text component, for cursor and position: relative you can add a className and change from here:

<Text className="menu-item">.....